Yoast/YoastSEO.js
# Add an English text to the full text tests.

## Summary

This PR can be summarized in the following changelog entry:

* no user-facing changes

## Relevant technical choices:

* Adds another English text to the full text tests

## Test instructions

This PR can be tested by following these steps:

* Run yarn test on the stories branch and make sure all checks pass


Requires #1644 Requires #1656
